updated_main.py

```diff
@@ -3,57 +3,86 @@
 import bs4
 import time
 import json
+import os
+from email.message import EmailMessage
+import ssl
+import smtplib
 
 # Running a for loop to get the latest price of the GBP
 for i in range(100):
 
     # Now extracting the value of British Pound in Indian Rupees
-    currency = "convert 1 pound to rs"
+    def Getting_current_price():
+        currency = "convert 1 pound to rs"
+        url = "https://google.com/search?q=" + currency
+        request_result = requests.get(url)
```

Comment:
```suggestion
        request_result = requests.get(f"https://google.com/search?q={currency}")
```
There is no need of this temporary variable. 

Rule of thumb is:
* If variable is used only once then don't make temporary variable.
* If some piece of code is used more than once then extract it out into variable/function.